sites/www.planetfour.org.conf

```diff
@@ -1,14 +1,28 @@
 server {
     include /etc/nginx/ssl.default.conf;
-    server_name www.planetfour.org;
+    server_name planetfour.org;
```

Comment:
This will either need to stay as it was, or have both domains:
```suggestion
    server_name www.planetfour.org planetfour.org;
```

But if it includes `planetfour.org` you will also need to remove the [existing redirect](https://github.com/zooniverse/static/blob/be5cd45468139bbf4e11f2247b6bfbbbc4430d38/nginx-redirects.conf#L50) in `nginx-redirects.conf`.

Reply:
The current version doesn't redirect `www.planetfour.org` as intended:

```
$ curl -v -H "Host: www.planetfour.org" localhost:8080
* Rebuilt URL to: localhost:8080/
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> GET / HTTP/1.1
> Host: www.planetfour.org
> User-Agent: curl/7.54.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Server: nginx/1.4.6 (Ubuntu)
< Date: Tue, 24 Mar 2020 14:51:12 GMT
< Content-Type: text/html
< Content-Length: 10878
< Connection: keep-alive
< x-amz-id-2: ik7Emw8w7X3J3WU+wHtC7hFgXryyrQxEfmmjEHcgJI6RpJAJ9lzrk9DWGXUKqshNUeM/gDHR6D0=
< x-amz-request-id: E34E73DE03A1B908
< Last-Modified: Thu, 02 May 2019 21:41:32 GMT
< x-amz-version-id: NttncZbkltf8_hR0lqDSpaZxHtUTFVqE
< ETag: "d42002638104e38aaf2741c31d459318"
< Access-Control-Allow-Origin: *
< Access-Control-Allow-Credentials: true
< Access-Control-Allow-Methods: GET
< Access-Control-Allow-Headers: DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type
< X-Content-Type-Options: nosniff
< Content-Security-Policy: frame-ancestors 'self' 
< X-Content-Security-Policy: frame-ancestors 'self' 
< X-XSS-Protection: 1; mode=block
< 
<!doctype html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang=""> <!--<![endif]-->
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title>Planet Four: Archive</title>
...snip...
✔ ~/Code/static [p4-relaunch {origin/p4-relaunch}|✚ 1] 
14:51 $ curl -v -H "Host: planetfour.org" localhost:8080
* Rebuilt URL to: localhost:8080/
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> GET / HTTP/1.1
> Host: planetfour.org
> User-Agent: curl/7.54.0
> Accept: */*
> 
< HTTP/1.1 301 Moved Permanently
< Server: nginx/1.4.6 (Ubuntu)
< Date: Tue, 24 Mar 2020 14:51:45 GMT
< Content-Type: text/html
< Content-Length: 193
< Connection: keep-alive
< Location: https://www.zooniverse.org/projects/mschwamb/planet-four
< 
<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.4.6 (Ubuntu)</center>
</body>
</html>
```